js/src/components/paid-ads/ads-campaign.js

```diff
@@ -91,13 +115,23 @@ export default function AdsCampaign( {
 				countrySelectHelperText={ helperText }
 				formProps={ formContext }
 			/>
-			<BudgetSection
-				formProps={ formContext }
-				disabled={ disabledBudgetSection }
-				countryCodes={ formContext.values.countryCodes }
-			>
-				<CampaignPreviewCard />
-			</BudgetSection>
+			{ ! loading && (
+				<BudgetSection
+					formProps={ formContext }
+					disabled={ disabledBudgetSection }
+					countryCodes={ formContext.values.countryCodes }
+					country={ country }
```

Comment:
@ankitrox We are passing those new props `country`, `dailyBudget` and `isMultiple` to `BudgetSection`. However those are being then passed to `BudgetRecommendation`. Can't we get those values directly in `BudgetRecommendation`?